ddtrace/tracer/span_msgp.go

```diff
@@ -267,15 +273,16 @@ func (z *span) DecodeMsg(dc *msgp.Reader) (err error) {
 // EncodeMsg implements msgp.Encodable
 func (z *span) EncodeMsg(en *msgp.Writer) (err error) {
 	// omitempty: check for empty values
-	zb0001Len := uint32(13)
-	var zb0001Mask uint16 /* 13 bits */
+	zb0001Len := uint32(14)
+	var zb0001Mask uint16 /* 14 bits */
+	_ = zb0001Mask
```

Comment:
Wow this does not look one bit readable... That assignment to `_` also looks... unnecessary?

Reply:
The files ending in `_msgp.go` are generated files so :(

Reply:
Ah okay.... Well that is weird.